broker/src/test/java/io/camunda/zeebe/broker/engine/RecordVersionTest.java

```diff
@@ -68,7 +68,7 @@ public void processInstanceRecordsShouldHaveBrokerVersion() {
     final var processInstanceKey = createProcessInstance(PROCESS_ID);
 
     // then
-    assertThat(RecordingExporter.records().limitToProcessInstance(processInstanceKey))
+    assertThat(RecordingExporter.records().betweenProcessInstance(processInstanceKey))
```

Comment:
:x: I find this naming confusing. Maybe this could be rewritten to
`betweenProcessInstanceEvents(processInstanceKey, List.of(ProcessInstanceIntent.ELEMENT_ACTIVATING), List.of(ProcessInstanceEvent.ELEMENT_COMPLETED, ProcessInstanceEvent.ELEMENT_TERMINATED))`

Although _events_ is also ambiguous. I am looking for a word that describes a point in the lifecycle of a process instance.

Reply:
I accept your objections regarding the naming but I don't see a reason to block this PR. We can change the name later if we have a better idea.

Reply:
Then let's create an issue for the renaming, so we don't forget it. And then this PR can go ahead.

Reply:
I will not create an issue for the renaming. If someone has a better idea for the name then we can just create a PR. I don't see that this name is so important.